docs/UserGuide.md

```diff
@@ -246,7 +248,7 @@ Finds persons whose names and module-role pairs contain any combination of the g
 
 Format: `find (n/KEYWORD)+ (r/KEYWORD)+`
 
-* Person matching at least one name keyword AND one module-role keyword will be returned (i.e. AND search).
+* Person matching at least one name keyword AND at least one module-role keyword will be returned (i.e. AND search).
```

Comment:
```suggestion
* Person matching at least one name keyword (if provided) AND at least one module-role keyword (if provided) will be returned (i.e. AND search).
```

in the case when `r/` is not provided, then there is no check. not sure if this wording is the best but need to somehow mention this